csharp/Adapter/Microsoft.Spark.CSharp/Services/LoggerServiceFactory.cs

```diff
@@ -31,7 +30,21 @@ public static void SetLoggerService(ILoggerService loggerServiceOverride)
         /// <returns>An instance of logger service</returns>
         public static ILoggerService GetLogger(Type type)
         {
-            return  loggerService.GetLoggerInstance(type);
+            return loggerService.Value.GetLoggerInstance(type);
+        }
+
+        /// <summary>
+        /// if there exists exe.config file, then use log4net
+        /// </summary>
+        /// <returns></returns>
+        private static ILoggerService GetDefaultLogger()
+        {
+            if (File.Exists(AppDomain.CurrentDomain.SetupInformation.ConfigurationFile))
```

Comment:
I thought but want to warn the user with the message "log4net:ERROR Failed to find configuration section 'log4net'", so I didn't add this check.
However, through these two days' usage, I found sometime if I don't want to use logger this warning seems ugly and It's boring to eliminate it (remove the App.config or add the log4net settings in App.config)
Because Visual Studio adds a default file of App.Config (which eventually will be deployed to xxx.exe.config) .

I'll add this check now.